docs/instrumentation-list.yaml

```diff
@@ -239,13 +239,21 @@ libraries:
       - org.apache.camel:camel-core:[2.19,3)
   cassandra:
   - name: cassandra-4.0
+    description: |
+      Instruments the Cassandra database client, providing database client spans and metrics for Cassandra queries.
     source_path: instrumentation/cassandra/cassandra-4.0
     scope:
       name: io.opentelemetry.cassandra-4.0
     target_versions:
       javaagent:
       - com.datastax.oss:java-driver-core:[4.0,4.4)
+    configurations:
+    - name: otel.instrumentation.common.db-statement-sanitizer.enabled
+      description: Enables or disables statement sanitization for database queries.
```

Comment:
wondering whether it would be better to use just `Enables statement sanitization for database queries.`